### DISCUSSION TITLE
发现开启mux后，设置bufferSize可以提高访问体验

### DISCUSSION THREAD

**Discussion by wenzhuning** [XTLS/Xray-core#1499] [General]
开启mux的可以试试，我测试设置了bufferSize为2048，加载数据明显变快，而且可以降低卡顿和阻塞感知，内存足的强烈建议设置bufferSize

**Comment by cross-hello** [comment]
תודה אתה

Jan 8, 2023 18:38:04 wenzn ***@***.***>:

**Comment by cross-hello** [comment]
We adjust it to 8096. Memory of the phone maybe could be slightly more used in the way.

Jan 8, 2023 18:38:04 wenzn ***@***.***>:
